src/components/collections/Editor/plugins/ComponentPickerPlugin/index.tsx

```diff
@@ -121,10 +124,13 @@ const getBaseOptions = (editor: LexicalEditor): ComponentPickerOption[] => [
           });
         }
       }),
-  }),
+  })
 ]
```

Comment:
Do we need to return an array here? if it's just one element, we can return the aiPromptOption object directly.

```suggestion
const aiPromptOption = (editor: LexicalEditor): ComponentPickerOption =>
  new ComponentPickerOption('AI Prompt', {
    icon: <SparklesIcon className="icon paragraph" />,
    keywords: ['ai', 'prompt', 'aiprompt'],
    onSelect: (): void =>
      editor.update(() => {
        const selection = $getSelection();
        if ($isRangeSelection(selection)) {
          $setBlocksType(selection, () => {
            const paragraph = $createParagraphNode();
            const prompt = $createAiPromptNode('Generating prompt with AI, please wait ...');
            paragraph.append(prompt);
            return paragraph;
          });
        }
      }),
  })
```